hbase-server/src/main/java/org/apache/hadoop/hbase/master/HbckChore.java

```diff
@@ -47,11 +47,11 @@
  */
 @InterfaceAudience.Private
 @InterfaceStability.Evolving
-public class HbckChecker extends ScheduledChore {
-  private static final Logger LOG = LoggerFactory.getLogger(HbckChecker.class.getName());
+public class HbckChore extends ScheduledChore {
+  private static final Logger LOG = LoggerFactory.getLogger(HbckChore.class.getName());
 
-  private static final String HBCK_CHECKER_INTERVAL = "hbase.master.hbck.checker.interval";
-  private static final int DEFAULT_HBCK_CHECKER_INTERVAL = 60 * 60 * 1000;
+  private static final String HBCK_CHORE_INTERVAL = "hbase.master.hbck.chore.interval";
+  private static final int DEFAULT_HBCK_CHORE_INTERVAL = 60 * 60 * 1000;
```

Comment:
Perhaps go back to the issue that added hbck checker and point at this one where we rename the class? In the releaasee note, do we list these configs? If so, note the names have changed? Otherwise, ignore this comment.

Reply:
Updated HBASE-22709's RN.